ner4opt/constants.py

```diff
@@ -0,0 +1,107 @@
+class Constants:
+    """Class for all constants."""
+
+    MODELS_DIRECTORY = 'models'
+
+    LEXICAL = 'lexical'
+    LEXICAL_PLUS = 'lexical_plus'
+    SEMANTIC = 'semantic'
+    HYBRID = 'hybrid'
+
+    # CRF Models
+    LEXICAL_MODEL_NAME = 'lexical.pkl'
+    LEXICAL_PLUS_MODEL_NAME = 'lexical_plus.pkl'
+    HYBRID_MODEL_NAME = 'hybrid.pkl'
+
+    # Deep Models
+    # Roberta Model
+    SEMANTIC_DEEP_MODEL = 'skadio/ner4opt-roberta-v1'
+    # Roberta Model trained as part of our Hybridization experiment
+    HYBRID_DEEP_MODEL = 'skadio/ner4opt-roberta-v2'
```

Comment:
I am confused with this naming.. The keyword "hybrid" is not referring to our "Hybrid" model, is it?

Reply:
.. hmm .. is this referring to the fine-tuned roberta?? IF SO, consider SEMANTIC_ROBERTA_MODEL and SEMANTIC_ROBERTA_FINE_TUNED_MODEL

Reply:
I changed naming conventions. The difference in the models is just due to their inherent randomness, the v2 model is the model which got trained as a part of our hybrid modeling experiment, seems to produce slightly better results. V1 model is an older model we trained, we submitted this model to the leaderboard too, but this didn't change our best scores. In the paper, we refer to that as just RoBERTa